Mathlib/CategoryTheory/Elementwise.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+
+! This file was ported from Lean 3 source module category_theory.elementwise
+! leanprover-community/mathlib commit 70fd9563a21e7b963887c9360bd29b2393e6225a
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Tactic.Elementwise
+import Mathlib.CategoryTheory.ConcreteCategory.Basic
+
+/-!
+# Use the `elementwise` attribute to create applied versions of lemmas.
+
+Usually we would use `@[elementwise]` at the point of definition,
+however some early parts of the category theory library are imported by `tactic.elementwise`,
+so we need to add the attribute after the fact.
+-/
+
+
+/-! We now add some `elementwise` attributes to lemmas that were proved earlier. -/
+
+
+open CategoryTheory
+
+-- This list is incomplete, and it would probably be useful to add more.
+attribute [elementwise] iso.hom_inv_id iso.inv_hom_id is_iso.hom_inv_id is_iso.inv_hom_id
```

Comment:
Hm, I did a workaround by first removing `simp` and then re-add it using `elementwise (attr := simp)`, which then adds *all* the correct simp lemmas.

Reply:
Okay, fixed the linter option instead

Reply:
@fpvandoorn I know we talked about how you might fix the linter option or re-implement the attribute handling for `elementwise`/`reassoc`, so here's a ping since @javra modified ToAdditive

Reply:
Floris was explaining to me that `elementwise`/`reassoc` shouldn't be using `applyAttributes`, since it does some very ToAdditive-specific things (including applying `to_additive`!).

Reply:
Ah, makes sense, my changes to `applyAttributes` are probably still reasonable

Reply:
Yes, this change is still sensible. Note there will be a small conflict with #2908